README.md

```diff
@@ -1,2 +1,28 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
- [ ] I kindly suggest that you add the table of contents from the README file.

| Text  | Images |
| ------------- | ------------- |
| Your README file| ![README md - HelloWorld5 - Gitpod Code and 26 more pages - Personal - Microsoft_ Edge 2023-03-21 19-34-49](https://user-images.githubusercontent.com/52934128/226710207-1b7f0e81-11cd-4ad0-b9d2-78b1ef94e59b.gif) |
| Professional README file | ![Recording 2023-02-14 at 12 52 20 (5)](https://user-images.githubusercontent.com/83360636/218715072-2b2d3a49-8219-4939-b350-45cc950f7f7a.gif) |